### PR TITLE
Updates for up and coming changes to the core profile layer

### PR DESCRIPTION
**Pull request by jamezp**

### CODE REVIEW

File: wf-core-tck-runner/rest-tck/pom.xml [hunk 1]
```diff
@@ -197,26 +197,19 @@
...
                         <configuration>
-                            <jboss-home>${jboss.home}</jboss-home>
-                            <offline>true</offline>
-                            <stdout>target/cli.out</stdout>
-                            <scripts>
-                                <script>src/test/resources/configure-wildfly.cli</script>
-                            </scripts>
+                            <target>
+                                <mkdir dir="${jboss.home}/modules/system/layers/base/org/jboss/as/controller/main/dir/META-INF/services/"/>
+                                <copy verbose="true" todir="${jboss.home}/modules/system/layers/base/org/jboss/as/controller/main/dir/META-INF/services/">
```
Comment (bstansberry) [inline wf-core-tck-runner/rest-tck/pom.xml:209]: Note to self: the 'overwrite' attribute defaults to 'false' which is good; we don't want to overwrite the file if it exists, which is does in newer code.

File: wf-core-tck-runner/rest-tck/src/test/resources/tck-config.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+          level: !undefine
+      logger:
+        org.jboss.resteasy:
+          level: TRACE
+    undertow:
+      server:
+        default-server:
+          http-listener:
+            default:
+              disallowed-methods:
```
Comment (bstansberry) [inline wf-core-tck-runner/rest-tck/src/test/resources/tck-config.yml:15]: Please add a comment explaining why this is needed.

Comment (bstansberry) [reply wf-core-tck-runner/rest-tck/src/test/resources/tck-config.yml:15]: For the other ones it would be good too.

**Review by bstansberry — CHANGES_REQUESTED** [review]
@jamezp Minor comment request; otherwise LGTM.

File: wf-core-tck-runner/rest-tck/src/test/resources/tck-config.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+      # it easier to see why things have failed.
+      logger:
+        org.jboss.resteasy:
+          level: TRACE
+    undertow:
+      server:
+        default-server:
+          http-listener:
+            default:
+              # We need to all HTTP methods to be allowed. By default, TRACE is not allowed
```
Comment (bstansberry) [inline wf-core-tck-runner/rest-tck/src/test/resources/tck-config.yml:18]: @jamezp Is this alternate wording correct?

"# A TCK test needs all HTTP methods to be allowed.  By default, TRACE is not allowed."

Sorry to be nitpicky but I'm aiming for there to be no ambiguity in these yaml files as to whether a change is needed to allow the TCK to do things that the spec doesn't require an impl to provide OOTB, vs changes made for other reasons.

Comment (jamezp) [reply wf-core-tck-runner/rest-tck/src/test/resources/tck-config.yml]: That makes sense to me. I'll fix it now.

**Pull request merged**